README.md

```diff
@@ -75,8 +75,9 @@ We appreciate all contributions to improve MMDeploy. Please refer to [CONTRIBUTI
 
 ## Acknowledgement
```

Comment:
In my opinion, it could be simplified and only including names and links is enough.
Something like this:

## Acknowledgement
We would like to sincerely thank following teams for their generous and excellent contributions to [MMDeploy](https://github.com/open-mmlab/mmdeploy):
- [OpenPPL](https://openppl.ai/home)
- [OpenVINO](https://docs.openvino.ai/latest/index.html)

Reply:
Agree

Reply:
We would like to sincerely thank the following teams for their contributions to MMDeploy: